R/satterthwaite.R

```diff
@@ -288,33 +273,15 @@ h_df_md_from_1d <- function(object, contrast) {
 
 #' Calculation of Satterthwaite Degrees of Freedom for Multi-Dimensional Contrast
 #'
-#' @description `r lifecycle::badge("experimental")`
```

Comment:
please keep lifecycle badges, and make sure to have them for all new exported functions too